### PR TITLE
MOSIP-44448 and MOSIP-44401 : Changed the pom versions to 1.4.0-SNAPSHOT and added new scenarios

### PR DESCRIPTION
**Pull request by BhuvanaShreeBS**
MOSIP-44448 : Updated the existing DSL dependency version from 1.3.0-SNAPSHOT to 1.4.0-SNAPSHOT
MOSIP-44401 : Added 4 new scearios
MOSIP-44393 : removed the repetitive steps logging in report 

## Summary by CodeRabbit

* **Chores**
  * Bumped project and module versions to 1.4.0-SNAPSHOT across the repository.
* **Tests**
  * Expanded acceptance test suite with many new negative and edge-case scenarios covering invalid inputs, mapping/configuration errors, packet creation failures, and cleanup flows.
* **Bug Fixes**
  * Hardened retry and failure detection, improved logging, relaxed success checks for tolerant responses, and refined handling for email-like attribute updates.

### CODE REVIEW

**Review by jayesh12234 — APPROVED** [review]

**Review by mohanachandran-s — APPROVED** [review]

File: mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json [hunk 1]
```diff
@@ -30030,6 +30030,720 @@
...
+         "Action":"$$uin=e2e_getUINByRid($$rid)"
+      },
+      "Step-9":{
+         "Description":"Waits for given period in seconds",
+         "Input Parameters":"Time period in seconds to wait",
+         "Return Value":"NA",
+         "Action":"e2e_wait(90)"
+      },
+      "Step-10":{
+         "Description":"Updates Demo graphic details and biometric in the persona file",
```
[the remark this answers is not in the document]
Comment (BhuvanaShreeBS) [reply mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json]: fixed

File: mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json [hunk 2]
```diff
@@ -30030,6 +30030,720 @@
...
+         "Action":"e2e_checkStatus(PROCESSED/*PACKET_STATUS*/,$$rid)"
+      },
+      "Step-8":{
+         "Description":"Gets the UIN for the given RID",
+         "Input Parameters":"RID",
+         "Return Value":"UIN",
+         "Action":"$$uin=e2e_getUINByRid($$rid)"
+      },
+      "Step-9":{
+         "Description":"Gets Email ID configuired for the UIN",
```
[the remark this answers is not in the document]
Comment (BhuvanaShreeBS) [reply mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json]: fixed

**Review by jayesh12234 — APPROVED** [review]

File: mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json [hunk 3]
```diff
@@ -30030,6 +30030,720 @@
          "Action":"e2e_DeletePacketData()"
       }
    },
+   {
+      "Scenario":"229",
+      "Tag":"Negative_Test",
+      "Persona":"ResidentMaleAdult",
+      "Group":"NA",
+      "Description":"Resident walks into the registration center completes the process with gender marked as invalid and failed to gets the UIN card.",
```
[the remark this answers is not in the document]
Comment (BhuvanaShreeBS) [reply mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json]: fixed

File: mosip-acceptance-tests/ivv-orchestrator/src/main/java/io/mosip/testrig/dslrig/ivv/orchestrator/PacketUtility.java [hunk 4]
```diff
@@ -693,10 +697,11 @@ public String getRID(String url, String packetPath, String residentPath, String
...
 		do {
-	  		count++;
+			count++;
 			Response response = postRequest(url, jsonReq.toString(), "Generate And UploadPacket", step);
-			
-			if (invalidMachineFlag.contentEquals("invalidMachine") && response.getBody().asString().toLowerCase().contains("failed to sign data")) {
+
+			if (invalidMachineFlag.contentEquals("invalidMachine")
+					&& response.getBody().asString().toLowerCase().contains("failed to sign data")) {
 				return "";
```
[the remark this answers is not in the document]
Comment (BhuvanaShreeBS) [reply mosip-acceptance-tests/ivv-orchestrator/src/main/java/io/mosip/testrig/dslrig/ivv/orchestrator/PacketUtility.java]: updated as per suggestion

File: mosip-acceptance-tests/ivv-orchestrator/src/main/java/io/mosip/testrig/dslrig/ivv/orchestrator/PacketUtility.java [hunk 5]
```diff
@@ -969,20 +979,20 @@ else if (supervOpertoDetails[3].equalsIgnoreCase(INVALID))
...
+				Map<String, String> result = machineHelper.getIdAndRegCenterIdByPublicKey(publicKey);
+				if (result != null) {
+					jsonReq.put("mosip.test.regclient.machineid", result.get("id"));
+					jsonReq.put("mosip.test.regclient.centerid", result.get("regCenterId"));
+				} else {
+					System.out.println("No matching machine found for public key: " + publicKey);
+				}
+			} catch (RigInternalError e) {
+				e.printStackTrace();
+			}
```
[the remark this answers is not in the document]
Comment (BhuvanaShreeBS) [reply mosip-acceptance-tests/ivv-orchestrator/src/main/java/io/mosip/testrig/dslrig/ivv/orchestrator/PacketUtility.java]: fixed

File: mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json [hunk 6]
```diff
@@ -30030,6 +30030,720 @@
...
+         "Input Parameters":"Time period in seconds to wait",
+         "Return Value":"NA",
+         "Action":"e2e_wait(90)"
+      },
+      "Step-12":{
+         "Description":"Updates Demographic details and biometric in the persona file",
+         "Input Parameters":"Data to update and persona file path. Other details can be found in the parameter in-line comments",
+         "Return Value":"NA",
+         "Action":"e2e_updateDemoOrBioDetails(leftIris,0/*MISS_FIELDS*/,0/*UPDATE_ATTRIBUTES*/,$$personaFilePath)"
+      },
```
[the remark this answers is not in the document]
Comment (BhuvanaShreeBS) [reply mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json]: yes changes done

File: mosip-acceptance-tests/ivv-orchestrator/src/main/java/io/mosip/testrig/dslrig/ivv/orchestrator/PacketUtility.java [hunk 7]
```diff
@@ -1050,84 +1061,85 @@ public String updateDemoOrBioDetail(String resFilePath, String testPersona,  Lis
...
-
-							this.hasError = true;
-							throw new RigInternalError("LangCode is missing in paramter");
 
+		// Handle the special case for "updateLargeDocInPersona"
+		if (step.getParameters().size() > 2
+				&& step.getParameters().get(2).equalsIgnoreCase("updateLargeDocInPersona")) {
+			String docPathValue = "yourDocPathValue";
+			String poaValue = String.format("{\"typeCode\": \"RNC\",\"docPath\": \"%s\"}", docPathValue);
+			updateAttribute.put("POA", poaValue);
```
[the remark this answers is not in the document]
Comment (BhuvanaShreeBS) [reply mosip-acceptance-tests/ivv-orchestrator/src/main/java/io/mosip/testrig/dslrig/ivv/orchestrator/PacketUtility.java]: Not required , continuing with current implementation

File: mosip-acceptance-tests/ivv-orchestrator/src/main/java/io/mosip/testrig/dslrig/ivv/orchestrator/PacketUtility.java [hunk 8]
```diff
@@ -1050,84 +1061,85 @@ public String updateDemoOrBioDetail(String resFilePath, String testPersona,  Lis
...
+				if (!allSuccess) {
+					this.hasError = true;
+					throw new RigInternalError("One or more result entries failed for attributes: " + attributeList);
+				}
+			}
+			// Case: Only a basic success string/object
+			else if (!body.contains("sucess")) {
+				this.hasError = true;
+				throw new RigInternalError("Response does not contain success status for attributes: " + attributeList);
+			}
```
[the remark this answers is not in the document]
Comment (BhuvanaShreeBS) [reply mosip-acceptance-tests/ivv-orchestrator/src/main/java/io/mosip/testrig/dslrig/ivv/orchestrator/PacketUtility.java]: Fixed as per suggestion

File: mosip-acceptance-tests/ivv-orchestrator/src/main/java/io/mosip/testrig/dslrig/ivv/orchestrator/PacketUtility.java [hunk 9]
```diff
@@ -1211,29 +1223,37 @@ public void bioAuth(String modility, String bioValue, String uin, Properties dev
...
-		input = JsonPrecondtion.parseAndReturnJsonContent(input, deviceProps.getProperty(BIOTYPE), "identityRequest."+BIOTYPE);
-		logger.info("After BioSubType: "+ input);
-		input = JsonPrecondtion.parseAndReturnJsonContent(input, deviceProps.getProperty(DEVICECODE), "identityRequest."+DEVICECODE);
+		logger.info("After UIN: " + input);
+		input = JsonPrecondtion.parseAndReturnJsonContent(input, deviceProps.getProperty(BIOSUBTYPE),
+				"identityRequest." + BIOSUBTYPE);
+		logger.info("After BioSubType: " + input);
+		input = JsonPrecondtion.parseAndReturnJsonContent(input, deviceProps.getProperty(BIOTYPE),
+				"identityRequest." + BIOTYPE);
+		logger.info("After BioSubType: " + input);
```
[the remark this answers is not in the document]
Comment (BhuvanaShreeBS) [reply mosip-acceptance-tests/ivv-orchestrator/src/main/java/io/mosip/testrig/dslrig/ivv/orchestrator/PacketUtility.java]: Continuing with current implementation

File: mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json [hunk 10]
```diff
@@ -30030,6 +30030,720 @@
...
+         "Input Parameters":"NA",
+         "Return Value":"NA",
+         "Action":"e2e_DeletePacketData()"
+      }
+   },
+   {
+      "Scenario":"231",
+      "Tag":"Positive_Test",
+      "Persona":"ResidentMaleAdult",
+      "Group":"",
```
[the remark this answers is not in the document]
Comment (BhuvanaShreeBS) [reply mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json]: fixed

File: mosip-acceptance-tests/ivv-orchestrator/src/main/java/io/mosip/testrig/dslrig/ivv/orchestrator/PacketUtility.java [hunk 11]
```diff
@@ -1079,10 +1080,12 @@ public String updateDemoOrBioDetail(String resFilePath, String testPersona,  Lis
...
-						                        ? (arr[1].trim().equalsIgnoreCase("testmosip")
-						                                ? "dslautomation@mosip.io"
-						                                : arr[1].trim() + "@mosip.io")
-						                        : arr[1].trim())));
+						                	    ? (arr[1].trim().equalsIgnoreCase("testmosip")
+						                	            ? "dslautomation@mosip.io"
+						                	            : (arr[1].toLowerCase().contains("invalid")
+						                	                    ? arr[1].trim() 
+						                	                    : arr[1].trim() + "@mosip.io"))
+						                	    : arr[1].trim())));
```
[the remark this answers is not in the document]
Comment (BhuvanaShreeBS) [reply mosip-acceptance-tests/ivv-orchestrator/src/main/java/io/mosip/testrig/dslrig/ivv/orchestrator/PacketUtility.java:1088]: Invalid email test cases are handled by directly passing invalid values from test data, allowing validation to fail as expected.

**Review by mohanachandran-s — APPROVED** [review]

**Review by mohanachandran-s — APPROVED** [review]

**Review by mohanachandran-s — APPROVED** [review]
